src/hooks/use-scroll-percentage/helpers/is-browser.ts

```diff
@@ -0,0 +1,7 @@
+/**
+ * Return `true` if `window` is defined,
+ * `false` otherwise.
+ */
+export function isBrowser(): boolean {
```

Comment:
A tiny helper that felt worth splitting out, maybe later worth elevating if it feels like it makes sense.